detectable/src/test/java/com/synopsys/integration/detectable/detectables/pnpm/unit/PnpmYamlTransformerTest.java

```diff
@@ -80,14 +86,14 @@ public void testThrowExceptionIfNoDirectDepenciesInLockFile() {
         pnpmLockYaml.devDependencies = null;
         pnpmLockYaml.optionalDependencies = null;
         try {
-            pnpmTransformer.generateCodeLocation(pnpmLockYaml, Arrays.asList(DependencyType.APP, DependencyType.DEV, DependencyType.OPTIONAL), projectNameVersion);
+            pnpmTransformer.generateCodeLocation(pnpmLockYaml, Arrays.asList(DependencyType.APP, DependencyType.DEV, DependencyType.OPTIONAL), projectNameVersion, linkedPackageResolver);
         } catch (IntegrationException e) {
         }
```

Comment:
I'm not sure how this will test that an exception is thrown if no assertions are made and the exception is caught. 

I recommend using Junit's `Assertions::assertThrows`
```
Assertions.assertThrows(IntegrationException.class,
    () -> pnpmTransformer.generateCodeLocation(pnpmLockYaml, Arrays.asList(DependencyType.APP, DependencyType.DEV, DependencyType.OPTIONAL), projectNameVersion, linkedPackageResolver)
);
```